CMakeLists.txt

```diff
@@ -140,16 +140,12 @@ if(COMPILER_SUPPORTS_MARCH_NATIVE)
     set(MARCH_NATIVE_OPT ON)
 endif()
 
-if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm64|aarch64.*|AARCH64.*")
- set(AARCH64 TRUE)
-endif()
+message("CMAKE_SYSTEM_PROCESSOR=<${CMAKE_SYSTEM_PROCESSOR}>")
 
-if(AARCH64)
+# Flags for SIMD support
+set(HAVE_NEON FALSE)
```

Comment:
I didn't test the ARM platform but shouldn't this `set()` be removed? I think it may interfere with line [224](https://github.com/tesseract-ocr/tesseract/pull/3484/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR224).

Reply:
You are right, thank you. Fixed in commit e0af8d12e6c8a58cda998cff975dc97bb63e425a.